apps/dolly-frontend/src/main/js/src/components/fagsystem/dokarkiv/visning/DokarkivVisning.tsx

```diff
@@ -22,12 +22,16 @@ export default ({ journalpost, miljoe }: Props) => (
 		<TitleValue title="Tittel" value={journalpost.tittel} />
 		<TitleValue title="Kanal" value={journalpost.kanal} />
 		<TitleValue title="Tema" value={journalpost.tema} />
-		<TitleValue title="Journalførende enhet" value={journalpost.journalfoerendeEnhet} />
-		<TitleValue title="Sakstype" value={showLabel('sakstype', journalpost.sak?.sakstype)} />
+		<TitleValue
+			title="Behandlingstema"
+			value={`${journalpost.behandlingstemanavn} (${journalpost.behandlingstema})`}
```

Comment:
`journalpost.behandlingstemanavn` viser `null` når jeg tester. Ser ut som det egentlig skal være `journalpost.temanavn`, for der har vi riktig verdi, eller kan dette variere? 🤔 Ser `behandlingstemanavn` er brukt flere steder i koden her da, så...

Reply:
Åja merkelig, temanavn er noe annet vi har hatt fra tidligere ja, det skal være behandlingstemanavn (f.eks: AA-Registeret) som beskriver behandlingstema kode (aa012345) 🤔 Det funka på freddan, men da må jeg prøve igjen og se hvorfor den ikke kommer med..